modules/distribution/src/repository/resources/conf/default.json

```diff
@@ -363,7 +363,135 @@
   "identity.events.schemes.OIDCLogoutHandler.properties.enable": false,
   "identity.events.schemes.confirmationCodesInvalidate.subscriptions": [
     "POST_DELETE_USER"
-  ]
+  ],
+
+
+
+  "identity.datasource": "jdbc/WSO2CarbonDB",
```

Comment:
this is not needed, we are using a new database now named WSO2IdentityDB, and we should not be giving the option to change it. (users may still point all to the same DB at TOML)